src/NServiceBus.Msmq.AcceptanceTests/When_worker_sends_a_message_for_delayed_retry.cs

```diff
@@ -0,0 +1,115 @@
+﻿namespace NServiceBus.AcceptanceTests.Routing
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using AcceptanceTesting.Customization;
+    using AcceptanceTesting.Support;
+    using Config;
+    using Features;
+    using NServiceBus.Routing.Legacy;
+    using NUnit.Framework;
+    using ObjectBuilder;
+    using Transport;
+
+    public class When_worker_sends_a_message_for_delayed_retry : NServiceBusAcceptanceTest
```

Comment:
shouldn't this test fail on your branch @danielmarbach ?
not sure whether that test is actually valid in v6: https://github.com/Particular/NServiceBus/issues/4325#issuecomment-262748735